### DISCUSSION TITLE
Feature request: Cookie Helper

### DISCUSSION THREAD

**Discussion by FraBle** [magne4000/universal-middleware#33] [Ideas]
Many web frameworks have an abstraction to handle common interactions with cookie management. For example, Hono has the [Cookie Helper](https://hono.dev/docs/helpers/cookie), and NextJS has the [`cookies` function](https://nextjs.org/docs/app/api-reference/functions/cookies). It would be beneficial to have something similar in `universal-middleware`.

**Comment by magne4000** [comment]
Released in `@universal-middleware/core@0.2.11`.
Documentation: [Cookie helpers](https://universal-middleware.dev/helpers/cookie)
